src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/ExceptionHandling.cs

```diff
@@ -244,13 +311,25 @@ private enum RhEHFrameType
                 isFirstRethrowFrame = false;
                 isFirstFrame = false;
             }
+#else
+#pragma warning disable CS8500
```

Comment:
That doesn't compile:
`error CS0306: The type 'EH.ExInfo' may not be used as a type argument`. I guess it is because of the `object _exception` member of the struct.

Reply:
It is because of it is a `ref struct`.